app/users/controllers.py

```diff
@@ -58,3 +58,17 @@ def login_user(credentials):
 		connect.unbind_s()
 		emit('auth', {'error': "Authentication error."})
 		return;
+
+@socketio.on('search_users')
+def login_user(search_string):
```

Comment:
The function above this has the same name, it's confusing and should be changed.